src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.test.tsx

```diff
@@ -53,6 +54,11 @@ describe('<BrowserRegionEditor', () => {
     test('contains submit and close buttons', () => {
       expect(wrapper.find('button[type="submit"]')).toHaveLength(1);
     });
+
+    test('has an overlay on top when region field is active', () => {	
```

Comment:
> when region field is active
The test doesn't assert anything about whether region field is active; it just sets `isDisabled` to `true`. So "has an overlay on top when disabled" 😄

Reply:
✅